src/app/account/keys/user-key-actions.ts

```diff
@@ -1,31 +1,27 @@
 'use server'
 
 import { db } from '@/database'
-import { getUserIdByClerkId } from '@/server/actions/user-actions'
+import { siUser } from '@/server/queries'
 
 export const getMemberUserPublicKey = async (clerkId: string) => {
```

Comment:
this is dangerous, what would happen if user A called this endpoint with user B's clerkId?  Instead we should always lookup user info from Clerk directly, we have a few helpers for that, I used siUser since it'll throw if the user isn't found

Reply:
I forgot that all server actions are technically public endpoints 😭